tracking/api/serializers.py

```diff
@@ -0,0 +1,103 @@
+from rest_framework import serializers
+from django_filters import rest_framework as filters
+from tracking.metrics import is_inside_tehran, get_filtered_speed_records, get_filtered_acceleration_records, \
+    calculate_distance, calculate_distance_outside_tehran
+from tracking.models import Location
+
+
+class LocationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Location
+        fields = '__all__'
+
+
+class CarInformationSerializer(serializers.ModelSerializer):
+    is_tehran = serializers.SerializerMethodField()
+    is_speed_high = serializers.SerializerMethodField()
+    is_acceleration_high = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Location
+        fields = ('car_id', 'is_tehran', 'is_speed_high', 'is_acceleration_high', 'latitude', 'longitude', 'created_at')
+
+    def get_is_tehran(self, obj):
+        return is_inside_tehran(obj.latitude, obj.longitude)
+
+    def get_is_speed_high(self, obj):
+        if obj.speed >= 120:
+            return 'Speed Dangerous'
+        else:
+            return 'Speed Safe'
+
```

Comment:
1- use constant for static variables (return values & 120)
2- use boolean values for return
```
    def get_is_speed_high(self, obj):
        if obj.speed >= HIGH_SPEED_THRESHOLD:
            return True

        return False
```